### DISCUSSION TITLE
Shortcut to turns off the media presentation

### DISCUSSION THREAD

**Discussion by Uilsonjw** [sircharlo/meeting-media-manager#252] [Ideas] [answered]
Thanks for aded keyboard shortcuts for media presentation mode. Could you please adjust the use the shortcut not only to turns on but also to turns off the media presentation.

The reason why I'm asking to do it is becaouse somtimes the media presentation screen is opneing on the first monitor as for example in the notebook, and once everyting is behind this screen is hard to access the contrls to colose it. If we hava a sohortcut, we can close it witha the sortcut.

**Comment by sircharlo** [comment, marked as answer]
Alt-Z; already implemented. Shows and hides the presentation screen.

**Reply by Uilsonjw** [reply]
Thanks. I didn't know it.
